src/servicebus/azext_servicebus/_params.py

```diff
@@ -0,0 +1,408 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands.parameters import (tags_type, get_enum_type, resource_group_name_type, name_type)
+
+from azure.cli.command_modules.servicebus._validators import _validate_max_delivery_count, _validate_maxSizeInMegabytes,\
+    _validate_dead_lettering_on_message_expiration, _validate_requires_session, _validate_enable_batched_operations, \
+    _validate_support_ordering, _validate_enable_express, _validate_enable_partitioning,\
+    _validate_dead_lettering_on_message_expiration,\
+    _validate_requires_session, _validate_requiresDuplicateDetection, _validate_auto_delete_on_idle, \
+    _validate_duplicate_detection_history_time_window, _validate_default_message_time_to_live, _validate_lock_duration
+
+
+# pylint: disable=line-too-long
+def load_arguments(self, _):
+
+    # region - Namespace Create
+
+    with self.argument_context('sb namespace check_name_availability') as c:
+        c.argument('namespace_name', options_list=['--name'], required=True, help='name of the Namespace')
+
+
+    with self.argument_context('sb namespace create') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, required=True)
+        c.argument('namespace_name', options_list=['--name'], required=True, help='name of the Namespace')
+        c.argument('tags', options_list=['--tags', '-t'], arg_type=tags_type, help='tags for the namespace in '
+                                                                                   'Key value pair format')
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(['Basic', 'Standard', 'Premium']),
+                   help='Sku name and allowed values are, Basic, Standard, Premium')
```

Comment:
Since you're correctly using get_enum_type, you don't need to include the help text `and allowed values are, Basic, Standard, Premium`

Reply:
resolved